src/install/requirements_frontend.txt

```diff
@@ -16,6 +14,7 @@ python-dateutil==2.8.2
 si-prefix==1.2.2
 uwsgi==2.0.20
 virtualenv
+werkzeug==2.1.2
```

Comment:
Maybe add a comment above this line to prevent anyone from reordering this list?

Reply:
Good idea